siliconcompiler/report/streamlit_viewer.py

```diff
@@ -575,12 +576,62 @@ def show_title_and_runs(title_col_width=0.7):
         streamlit.columns([title_col_width, 1 - title_col_width], gap="large")
 
     with title_col:
-        streamlit.title(f'{new_chip.design} dashboard', anchor=False)
+        streamlit.markdown(
+            '''
+            <head>
+                <style>
+                    .logo-container {
+                        display: flex;
+                        align-items: flex-start;
+                    }
+                    .logo-image {
+                        margin-right: 10px;
+                        margin-top: -10px;
+                    }
+                    .logo-text {
+                        display: flex;
+                        flex-direction: column;
+                        margin-top: 13px;
+                    }
+                    .text1 {
+                        color: #F1C437; /* Yellow color */
+                        font: IBM Plex Sans !important;
+                        font-weight:700 !important;
+                        font-size:30px !important;
+                        margin-bottom: -15px;
+                    }
+                    .text2 {
+                        color: #1D4482; /* Blue color */
+                        font: IBM Plex Sans !important;
+                        font-weight:700 !important;
+                        font-size:30px !important;
+                    }
+                </style>
+            </head>''',
+            unsafe_allow_html=True
```

Comment:
Why is it considered unsafe by streamlit?
Just style wise or something on top?

Reply:
Just looked it up. Streamlit cannot ensure that the HTML is not vulnerable to hacks.

Reply:
Oh I see. I think if you are passing in a static string that should be fine.